tutorials/06-physics-simulation-concepts.md

```diff
@@ -35,8 +35,8 @@ cars depending on physics engines and visualize the collision concept.
 To run the demo world, download the SDFormat file by:
 
 ```bash
-wget https://raw.githubusercontent.com/gazebosim/gz-sim/gz-physics7/examples/worlds/diff_drive.sdf -P ~
-wget https://raw.githubusercontent.com/gazebosim/gz-sim/gz-physics7/examples/worlds/velocity_control.sdf -P ~
+wget https://raw.githubusercontent.com/gazebosim/gz-sim/gz-physics8/examples/worlds/diff_drive.sdf -P ~
```

Comment:
I think we could either do `main` here or just stick with `gz-physics8` knowing the release branch will eventually be created next year before the release so we don't have to change it again. Either way works

Reply:
Yup, that was my logic as well. We'll have to change it back when we release, so I thought it's okay to not use `main`.